legacy/docker-compose.yml

```diff
@@ -568,80 +568,10 @@ services:
         image: "${DOCKER_USERNAME}/msvc15:${DOCKER_BUILD_TAG}"
         container_name: msvc15
         tty: true
-    android-clang8:
```

Comment:
Removed android because we no longer build it for CCI and not all folders are available, so docker-compose command fails to parse this file.